plugins/techdocs/src/Router.tsx

```diff
@@ -75,10 +77,16 @@ export const EmbeddedDocsRouter = (props: PropsWithChildren<{}>) => {
     },
   ]);
 
-  const projectId = entity.metadata.annotations?.[TECHDOCS_ANNOTATION];
+  const projectId =
+    entity.metadata.annotations?.[TECHDOCS_ANNOTATION] ||
+    entity.metadata.annotations?.[TECHDOCS_EXTERNAL_ANNOTATION];
 
   if (!projectId) {
-    return <MissingAnnotationEmptyState annotation={TECHDOCS_ANNOTATION} />;
+    return (
+      <MissingAnnotationEmptyState
+        annotation={[TECHDOCS_ANNOTATION, TECHDOCS_EXTERNAL_ANNOTATION]}
```

Comment:
Just noting that the messaging here will be a bit odd, it'll [tell the user](https://github.com/backstage/backstage/blob/12e081d5c51fb58929b6748f1b38abb01bf2025b/packages/core-components/src/components/EmptyState/MissingAnnotationEmptyState.tsx#L89) `You need to add` both those annotations. Maybe not much to do about that now

Reply:
Yup its something thats difficult maybe I remove it?

Reply:
figured remove it and let the doco work and not confuse people as the primary way of doing it will be the Techdocs not the entity